tests/php/Repository/TaxonomyRepositoryTest.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Bolt\Tests\Repository;
+
+use Bolt\Entity\Taxonomy;
+use Symfony\Bundle\FrameworkBundle\Test\KernelTestCase;
+
+class TaxonomyRepositoryTest extends KernelTestCase
```

Comment:
Because the taxonomyrepository doesn't have those _yet_. It'll surely get some specific methods soon, and then I'll add them to the tests. For now, I just wanted to get the skeleton of the test in, to force me to add more tests when applicable.

Reply:
If it will force you then fo ahead :) But at least add a @todo reminder

Reply:
Todo added!